src/gateway/constants.ts

```diff
@@ -0,0 +1,85 @@
+// https://discord.com/developers/docs/topics/opcodes-and-status-codes#gateway-gateway-opcodes
```

Comment:
Opcodes and Close codes are in `/src/util/Constants.ts`

Reply:
Done